kit-slides.typ

```diff
@@ -22,6 +22,8 @@
 #let yellow = rgb(252, 229, 0)
 #let orange = rgb(223, 155, 27)
 #let red = rgb(162, 34, 35)
+#let lightgreen = rgb(140, 182, 60)
+#let gray = rgb(0, 0, 0)
```

Comment:
Redefining gray as black will probably be confusing.

This theme already provides `black70`. Also `black` is already provided by typst itself.